src/sentry/models/exporteddata.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+from uuid import uuid4
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
```

Comment:
We typically organize our imports into 3 groups:
 - Core python modules
 - 3rd party modules
 - sentry modules